tools/hyperv_kdump_config.ps1

```diff
@@ -0,0 +1,166 @@
+﻿########################################################################
+#
+# Linux on Hyper-V and Azure Test Code, ver. 1.0.0
+# Copyright (c) Microsoft Corporation
+#
+# All rights reserved. 
+# Licensed under the Apache License, Version 2.0 (the ""License"");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0  
+#
+# THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION
+# ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR
+# PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
+#
+# See the Apache Version 2.0 License for specific language governing
+# permissions and limitations under the License.
+#
+########################################################################
+
+<#
+.Synopsis
+    Configured Kdump on Linux VMs running on Hyper-V.
+
+.Description
+    
+    Test parameters
+
+.Parameter vmName
+    Name of the VM to test.
+
+.Parameter  hvServer
+    Name of the Hyper-V server hosting the VM.
+
+.Parameter  vmRootPasswd
+    Root password of the VM.
+
+Please keep plink.exe, pscp.exe in bin\ folder where script is.
+#>
+
+#param ([String] $vmName, [String] $hvServer, [String] $vmRootPasswd)
+
+param(

```

Comment:
We should provide an optional IPaddress parameter , in case user wants to provide.